package.json

```diff
@@ -8,14 +8,14 @@
 		"prebuild": "npm run prebuild:pancake && node src/helper/helper.js components",
 		"build:site": "./node_modules/.bin/cuttlebelle",
 		"build:autoprefix": "postcss src/assets/css/style.css -u autoprefixer -r && postcss src/assets/css/example.css -u autoprefixer -r",
-		"build:sass": "mkdir -p src/assets/css/ && node-sass --output-style compressed src/sass/style.scss src/assets/css/style.css && node-sass --output-style compressed src/sass/example.scss src/assets/css/example.css",
-		"build:sass:dev": "mkdir -p src/assets/css/ && node-sass --output-style compact --source-map true src/sass/style.scss src/assets/css/style.css && node-sass --output-style compact --source-map true src/sass/example.scss src/assets/css/example.css",
+		"build:sass": "./node_modules/.bin/mkdirp src/assets/css/ && node-sass --output-style compressed src/sass/style.scss src/assets/css/style.css && node-sass --output-style compressed src/sass/example.scss src/assets/css/example.css",
+		"build:sass:dev": "./node_modules/.bin/mkdirp src/assets/css/ && node-sass --output-style compact --source-map true src/sass/style.scss src/assets/css/style.css && node-sass --output-style compact --source-map true src/sass/example.scss src/assets/css/example.css",
```

Comment:
```suggestion
		"build:sass:dev": "mkdirp src/assets/css/ && node-sass --output-style compact --source-map true src/sass/style.scss src/assets/css/style.css && node-sass --output-style compact --source-map true src/sass/example.scss src/assets/css/example.css",
```

Reply:
I don't think running `mkdirp` without NPM like that will work unless it's installed globally ( i.e npm i -g mkdirp ).

I think mkdirp was intended to be used as a library in src code anyway. We could introduce a `mkdirp` script that runs this command with npm ? Otherwise calling it directly with node_modules is the only way

Reply:
if it's installed as a dependency it should work, try it locally?

Reply:
works for me 😅